contracts/core/constants/Aave.sol

```diff
@@ -3,7 +3,15 @@ pragma solidity ^0.8.15;
 string constant AAVE_LENDING_POOL = "AaveLendingPool";
 string constant AAVE_WETH_GATEWAY = "AaveWethGateway";
 
-string constant BORROW_ACTION = "AaveBorrow";
-string constant DEPOSIT_ACTION = "AaveDeposit";
-string constant WITHDRAW_ACTION = "AaveWithdraw";
-string constant PAYBACK_ACTION = "AavePayback";
+/**
+ * @dev We do not include patch versions in contract names to allow
+ * for hotfixes of Action contracts
+ * and to limit updates to TheGraph
+ * if the types encoded in emitted events change then use a minor version and
+ * update the ServiceRegistry with a new entry
+ * and update TheGraph decoding accordingly
+ */
+string constant BORROW_ACTION = "AaveBorrow_2";
```

Comment:
Does that mean that if there are fixes then we will add the minor version? So for now it's `AaveBorrow_2 `and if there is a fix it will be `AaveBorrow_2_1` or `AaveBorrow_2_2`?